src/transaction_builder.h

```diff
@@ -86,7 +86,7 @@ class TransactionBuilder
         uint256 ovk,
         libzcash::SaplingPaymentAddress to,
         CAmount value,
-        std::array<unsigned char, ZC_MEMO_SIZE> memo);
+        std::array<unsigned char, ZC_MEMO_SIZE> memo = {{0xF6}});
```

Comment:
I would have expected this to cause the wallet_listreceived.py tests to fail, since this PR doesn't update them: https://github.com/zcash/zcash/blob/master/qa/rpc-tests/wallet_listreceived.py#L14-L16
Why is that not the case, and does it indicate a test coverage problem?

Reply:
Oh, this is probably an actual issue. We ran `try` before the rebase, when the above RPC test was not in master (and thus not in this branch). I'll test locally, and push a change.

Reply:
Yep, breaks as expected! Pushed a fix.